src/core/shape/attributes.js

```diff
@@ -348,10 +348,76 @@ p5.prototype.strokeJoin = function(join) {
  * </code>
  * </div>
  */
-p5.prototype.strokeWeight = function(w) {
+p5.prototype.strokeWeight = function (w) {
```

Comment:
A small note: try to avoid changing formatting of the source files where not necessary (usually due to some editors automatically formatting things) as that can add noise to the PR itself. Not a big deal here though.